docs/reference/search/search.asciidoc

```diff
@@ -50,3 +50,39 @@ Or even search across all indices and all types:
 $ curl -XGET 'http://localhost:9200/_search?q=tag:wow'
 --------------------------------------------------
 
+=== Fast check for any matching docs
+
+In case we only want to know if there are any documents matching a
+specific query, we can set the `size` to `0` to indicate that we are not
+interested in the search results. Also we can set `terminate_after` to `1`
+to indicate that the query execution can be terminated whenever the first
+matching document was found (per shard).
+
+[source,js]
+--------------------------------------------------
+$ curl -XGET 'http://localhost:9200/_search?q=tag:wow&size=0&terminate_after=1'
+--------------------------------------------------
+
+The response will not contain any hit, but what we have to pay attention to
+is the `total_hits` field that tells us how many documents matched the query,
+before the query was terminated. Also if the query was terminated early, the
+`terminated_early` flag will be set to `true` in the response.
+
```

Comment:
Maybe I'm being captain Obvious, but it could be worth just saying that a total hit count equal to 0 means that there are no matches, while a count greater than 0 means that there are matches?